sscanss/app/widgets/graphics.py

```diff
@@ -739,19 +1098,39 @@ def render(self, painter, rect):
             transform.rotate(angle)
             transform.translate(-center.x(), -center.y())
             rotated_point = transform.map(point)
-            painter.drawLine(center.x(), center.y(), rotated_point.x(), rotated_point.y())
+            painter.drawLine(QtCore.QLineF(center.x(), center.y(), rotated_point.x(), rotated_point.y()))
 
         self.center = center
 
     @staticmethod
     def toPolar(x, y):
+        """Converts point in cartesian coordinate to polar coordinates
+
+        :param x: x coordinate
+        :type x: float
+        :param y: y coordinate
+        :type y: float
+        :return: point in polar coordinates (angle in degrees)
+        :rtype: Tuple(float, float)
+        """
         radius = math.sqrt(x * x + y * y)
```

Comment:
Can be done with hypot `radius = math.hypot(x,y)`

Reply:
nice, I've never seen this function